src/rf/apps/workers/management/commands/start-worker-process.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.core.management.base import BaseCommand
+
+from apps.workers import process
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        p = process.QueueProcessor()
```

Comment:
I line or two to `self.stdout.write` here would be good if there isn't already a message inside `start_polling`. Just something to signal that the process has started.